docs/use_updownload.md

```diff
@@ -0,0 +1,3 @@
+# Up and download tool
```

Comment:
Leave it out and create a story to develop the missing parts.

Reply:
I added rudimentary info.